lametro/templates/lametro/board_members.html

```diff
@@ -19,6 +19,13 @@
 
 {% cache 86400 members_wrapper 'members' %}
 
+<div class="container-fluid">
+    <br>
+    <div class="alert alert-danger mt-3" role="alert">
```

Comment:
What do you think about using a less severe color scheme here. Maybe `alert-info`?

Reply:
Here's what `alert-info` looks like:
<img width="1435" alt="Screen Shot 2022-02-23 at 2 47 13 PM" src="https://user-images.githubusercontent.com/36973363/155396517-cebab152-b616-4327-8c21-826a7a149422.png">

I think it looks good. The links also are obvious, color-wise. Do we want to underline them also?